tasks/main.yml

```diff
@@ -1,4 +1,10 @@
 ---
+- name: Check Terraform Version
+  shell: terraform --version | grep {{ terraform_version }}
+  register: current_terraform_version
+  changed_when: current_terraform_version.rc != 0
```

Comment:
The shell command always registers as an ansible change, so the playbook's idempotence check fails (even though that command doesn't alter anything). I added the `changed_when` to remedy that. I'll add a comment to `main.yml` to clarify